tools/gluten-te/defaults.conf

```diff
@@ -27,6 +27,9 @@ DEFAULT_HTTP_PROXY_PORT=
 # If on, use maven mirror settings for PRC's network environment
 DEFAULT_USE_ALI_MAVEN_MIRROR=OFF
 
+# Choose operator system
+DEFAULT_OS_IMAGE=ubuntu:20.04
```

Comment:
We can somehow override all generated images with a changed name-suffix with its tag, e.g.

```
gluten-te/gluten-buildenv-ubuntu:20.04
```

So that the images can be permanently cached to speed-up builds.